src/util/ErrorHandler.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Custom ObjectTypeError which extends Error class.
+ */
+class ObjectTypeError extends Error {
```

Comment:
Why is this named `ErrorHandler.ts` when the class name is `ObjectTypeError`? Also, Can we not simply do `throw new Error()`?

Reply:
This file isn't needed.

Reply:
To give specific error type i.e: `ObjectTypeError` i created the file

Also, I feel this might not be needed.